ajax/libs/material-design-icons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "material-design-icons",
+  "version": "2.0.0",
+  "filename": "iconfont/MaterialIcons-Regular.ttf",
+  "description": "Material Design icons by Google",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/material-design-icons"
+  },
+  "keywords": [
+    "icons",
+    "material",
+    "material-design",
+    "google"
+  ],
+  "author": "Material Design Authors",
+  "license": "CC-BY-4.0",
+  "bugs": {
+    "url": "https://github.com/google/material-design-icons/issues"
+  },
+  "homepage": "https://github.com/google/material-design-icons",
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:google/material-design-icons.git",
+    "basePath": "/",
```

Comment:
I've changed it to `"basePath": "/iconfont/"`. I wasn't sure whether to do add it, as the material-design-icons also has a lot of sprite sheets in [/sprites/](https://github.com/google/material-design-icons/tree/master/sprites). I didn't want to add them as the iconfont is the primary method and the sprite sheets are both large in number and partially outdated, but leave the option open to include more files from different directories.